### PR TITLE
[webpack-config] Added lighthouse audits to CI

### PR DESCRIPTION
**Pull request by EvanBacon**
Run a lighthouse audit on the example Expo project that is built in production mode.

### CODE REVIEW

**Comment by EvanBacon** [comment]
Would probably make sense to implement a hook that published and tested on a staging URL. Puppeteer scores are swayed by the serve-cli

File: packages/webpack-config/tests/__tests__/basic-test.prod.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  for (const lighthouseTest of tests) {
+    const { minValue = 1.0 } = lighthouseTest;
+    it(
+      minValue >= 1.0
+        ? `should have a perfect ${lighthouseTest.name} score`
+        : `should have ${lighthouseTest.name} score of ${minValue * 100} or greater`,
+      async () => {
+        const score = await getLighthouseResultsAsync(lhr, lighthouseTest.field);
+        expect(score).toBeGreaterThanOrEqual(minValue);
+        expect(score).toMatchSnapshot();
```
Comment (ide) [inline packages/webpack-config/tests/__tests__/basic-test.prod.js]: keep an eye on this in case it gets annoying

**Comment by ide** [comment]
> Would probably make sense to implement a hook that published and tested on a staging URL. > Puppeteer scores are swayed by the serve-cli

What specifically do you mean by this? We probably do want a local server so that CI is shielded from external I/O (specifically networks and uptime).

**Comment by EvanBacon** [comment]
stale - we should use Lighthouse CI now

**Pull request closed without merging**